<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocialNetworkWindow</class>
 <widget class="QMainWindow" name="SocialNetworkWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SocialNetworkWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="profileWidget" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>-40</y>
      <width>711</width>
      <height>551</height>
     </rect>
    </property>
    <widget class="QLabel" name="profileLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>410</y>
       <width>191</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Profile</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="friendsTable">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>40</y>
       <width>381</width>
       <height>101</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="postsLabel">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>410</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextInteractionFlag::LinksAccessibleByMouse|Qt::TextInteractionFlag::TextSelectableByMouse</set>
     </property>
    </widget>
    <widget class="QPushButton" name="backToProfileButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>161</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Back To My Profile</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="suggestionsTable">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>251</width>
       <height>141</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="addFriendButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>350</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Friend</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addThisFriendButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>380</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Add This Friend</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addPostButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>510</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Post</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="postsScrollArea">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>170</y>
       <width>261</width>
       <height>201</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>259</width>
        <height>199</height>
       </rect>
      </property>
      <widget class="QWidget" name="postsContainer" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-10</y>
         <width>361</width>
         <height>101</height>
        </rect>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>301</width>
          <height>141</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="postsLayout"/>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QTextEdit" name="postInput">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>410</y>
       <width>271</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="loginWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>258</width>
      <height>259</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="loginPromptLabel">
       <property name="text">
        <string>Enter your name.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="nameInput">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loginButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>860</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
